sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/util/CosmosPagedFlux.java

```diff
@@ -65,12 +65,18 @@
     /**
      * Handle for invoking "side-effects" on each FeedResponse returned by CosmosPagedFlux
      *
-     * @param feedResponseConsumer handler
+     * @param newFeedResponseConsumer handler
      * @return CosmosPagedFlux instance with attached handler
      */
     @Beta(value = Beta.SinceVersion.V4_6_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
-    public CosmosPagedFlux<T> handle(Consumer<FeedResponse<T>> feedResponseConsumer) {
-        return new CosmosPagedFlux<T>(this.optionsFluxFunction, feedResponseConsumer);
+    public CosmosPagedFlux<T> handle(Consumer<FeedResponse<T>> newFeedResponseConsumer) {
```

Comment:
`handle()` API changes look good, thanks @FabianMeiswinkel . 
Please also port these changes to master branch.